src/test/scala/nl/knaw/dans/deposit/DepositPropertiesSpec.scala

```diff
@@ -50,6 +51,7 @@ class DepositPropertiesSpec extends TestSupportFixture
 
     props.identifier.doi.value shouldBe empty
     props.identifier.doi.registered.value shouldBe false
+    props.identifier.doi.action.value shouldBe create
```

Comment:
`create` -> `Action.create`? (same below)
in this case also don't forget to remove the import above